app/assets/javascripts/shared/SharedService.js.coffee

```diff
@@ -66,6 +66,18 @@ SharedService = ($http, $state, $window, $document) ->
     'dahlia.disclaimer':
       buildUrl: (state, params) -> Service._addLanguageAndParamsToUrl(params.lang, "/disclaimer")
       shouldRailsRoute: (isFirstLoad) -> !isFirstLoad
+    'dahlia.sign-in':
```

Comment:
question: This is not an "account information" page and doesn't use the same env var either. Should this change be in a separate PR / Jira ticket?

Reply:
Thanks for catching! I removed that line